ApplicationLibCode/FileInterface/RifEclipseSummaryAddress.cpp

```diff
@@ -466,21 +473,32 @@ std::string RifEclipseSummaryAddress::generateStringFromAddresses( const std::ve
 //--------------------------------------------------------------------------------------------------
 bool RifEclipseSummaryAddress::isDependentOnWellName( SummaryVarCategory category )
 {
-    // clang-format off
-    if (category == SUMMARY_WELL ||
-        category == SUMMARY_WELL_COMPLETION ||
-        category == SUMMARY_WELL_COMPLETION_LGR ||
-        category == SUMMARY_WELL_LGR ||
-        category == SUMMARY_WELL_SEGMENT)
+    if ( category == SummaryVarCategory::SUMMARY_WELL || category == SummaryVarCategory::SUMMARY_WELL_COMPLETION ||
+         category == SummaryVarCategory::SUMMARY_WELL_COMPLETION_LGR || category == SummaryVarCategory::SUMMARY_WELL_LGR ||
+         category == SummaryVarCategory::SUMMARY_WELL_SEGMENT )
     {
         return true;
     }
 
-    // clang-format on
-
     return false;
 }
```

Comment:
Unecessary branch? Just return the result of the if condition.

`    return ( category == SummaryVarCategory::SUMMARY_WELL || category == SummaryVarCategory::...);`